itou/prescribers/management/commands/sync_ft_organizations.py

```diff
@@ -0,0 +1,150 @@
+from django.db import IntegrityError, transaction
+from django.db.models import Q
+
+from itou.cities.models import City
+from itou.common_apps.address.models import BAN_API_RELIANCE_SCORE, lat_lon_to_coords
+from itou.prescribers.enums import PrescriberAuthorizationStatus, PrescriberOrganizationKind
+from itou.prescribers.models import PrescriberOrganization
+from itou.utils.apis import pole_emploi_api_client
+from itou.utils.command import BaseCommand
+from itou.utils.sync import DiffItemKind, yield_sync_diff
+
+
+def name_from_api_data(data):
+    return f'France Travail - {data["libelle"]}'
+
+
+def phone_from_api_data(data):
+    return data.get("contact", {}).get("telephonePublic", "")
+
+
+def email_from_api_data(data):
+    return data.get("contact", {}).get("email", "")
+
+
+def address_from_api_data(data):
+    return data["adressePrincipale"].get("ligne4")
+
+
+def address_extra_from_api_data(data):
+    return data["adressePrincipale"].get("ligne3", "")
+
+
+def post_code_from_api_data(data):
+    return data["adressePrincipale"].get("bureauDistributeur")
+
+
+def coordinates_from_api_data(data):
+    return lat_lon_to_coords(data["adressePrincipale"].get("gpsLat"), data["adressePrincipale"].get("gpsLon"))
+
+
+def insee_city_from_api_data(data):
+    try:
+        return City.objects.get(code_insee=data["adressePrincipale"].get("communeImplantation"))
+    except City.DoesNotExist:
+        return None
+
+
+def fill_organization_from_api_data(obj, siret, data):
+    # Basic information
+    obj.siret = siret
+    obj.name = name_from_api_data(data)
+    # Contact information
+    obj.phone = phone_from_api_data(data)
+    obj.email = email_from_api_data(data)
+    # Address
+    obj.address_line_1 = address_from_api_data(data)
+    obj.address_line_2 = address_extra_from_api_data(data)
+    obj.post_code = post_code_from_api_data(data)
+    # Geolocation
+    obj.coords = coordinates_from_api_data(data)
+    obj.geocoding_score = BAN_API_RELIANCE_SCORE if obj.coords else None
+    # INSEE city
+    obj.insee_city = insee_city_from_api_data(data)
+    obj.city = obj.insee_city.name if obj.insee_city else ""
+    obj.department = obj.insee_city.department if obj.insee_city else ""
+
+
+class Command(BaseCommand):
+    help = "Synchronize 'agences' informations from the FT API."
+
+    def add_arguments(self, parser):
+        parser.add_argument("action", choices=["update-information", "fix-empty-safir"])
+        parser.add_argument("--wet-run", dest="wet_run", action="store_true")
+
+    def fix_empty_safir(self, data, *, wet_run=False):
```

Comment:
Pas besoin de donner une valeur par défaut, c’est déjà gérer par le add_argument, et la signature de `handle` (qui est également inutile).
```suggestion
    def fix_empty_safir(self, data, *, wet_run):
```

Reply:
Noui :grin:, je part du principe que l'argument n'étant pas obligatoire pour le fonctionnement de la fonction il est donc optionnel et donc avec une valeur par défaut.